net/tstun/tun_features_linux.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package tstun
+
+import (
+	"os"
+	"strings"
+
+	"github.com/tailscale/wireguard-go/tun"
+)
+
+func setLinkFeatures(dev tun.Device) error {
+	val := os.Getenv("TS_TUN_DISABLE_UDP_GRO")
+	if strings.ToLower(val) == "true" || val == "1" {
```

Comment:
the envknob package has helpers for all of this, e.g.

```
disable := envknob.Bool("TS_TUN_DISABLE_UDP_GRO")
```

Reply:
updated